client/app/host/page.tsx

```diff
@@ -1,3 +1,368 @@
+                                                    <DropdownMenuPortal>
+                                                        <DropdownMenuSubContent>
+                                                            <DropdownMenuItem>
+                                                                Name
+                                                            </DropdownMenuItem>
+                                                            <DropdownMenuItem>
+                                                                Password
+                                                            </DropdownMenuItem>
+                                                            <DropdownMenuItem>
+                                                                Points
+                                                            </DropdownMenuItem>
+                                                        </DropdownMenuSubContent>
+                                                    </DropdownMenuPortal>
+                                                </DropdownMenuSub>
+                                                <DropdownMenuSeparator />
+                                                <DropdownMenuItem
+                                                    onClick={() => handleDisconnectTeam(team.name)}
+                                                >
+                                                    Kick
+                                                </DropdownMenuItem>
+                                            </div>
+                                        ) : (
+                                            <div>
+                                                <DropdownMenuItem
+                                                    onClick={() => handleRemoveTeam(team.name)}
+                                                >
+                                                    Delete
+                                                </DropdownMenuItem>
+                                            </div>
+                                        )}
+                                    </DropdownMenuContent>
+                                </DropdownMenu>
+                            </span>
+                        ))}
+                </div>
+                <Separator className="mt-2" />
+
+                <div className="mb-5 mt-auto flex flex-col items-center justify-center">
+                    <Separator className="mt-2" />
+                    <p className="mx-auto my-2.5 text-2xl uppercase">Server</p>
+                    <Button
+                        className={`h-fit w-fit px-5 py-3 text-2xl font-bold lowercase text-black ${isServerOn === null ? 'bg-gray-500' : isServerOn ? 'bg-red-600 hover:bg-red-500' : 'bg-green-600 hover:bg-green-500'}`}
+                        onClick={handleToggleServer}
+                    >
+                        {isServerOn === null ? 'loading...' : isServerOn ? 'stop' : 'start'}
```

Comment:
Should create an enum with the server's state that is something like `'loading' | 'stopped' | 'running'` and use that here instead of checking this nullable boolean.